No handlers could be found for logger "gensim.models.doc2vec"
main: loading grounder from file...
main: ... done
main: instantiating IO...
main: ... done
main: instantiating Agent...
main: ... done
main: running command dialog...
say_to_user_with_referents: What should I do? {}
get_from_user requested feedback
get_from_user (raw): 'Got into the office belonging to Robert "Bob"Brown.'
get_from_user (processed): 'got into the office belonging to robert bob"brown'
say_to_user: I haven't heard the word 'belonging' before. Does it refer to properties of things, like a color, shape, or weight?
get_from_user requested feedback
get_from_user (raw): 'No'
get_from_user (processed): 'no'
parse_and_ground_utterance: parsed 'no' to no
parse_and_ground_utterance: resulting groundings with normalized confidences: 
	no 1.0
parse_and_ground_utterance: could not generate a parse for the utterance
say_to_user_with_referents: You want me to go somewhere? (not manipulate any objects) {'action': 'walk'}
get_from_user requested feedback
get_from_user (raw): 'Yes'
get_from_user (processed): 'yes'
parse_and_ground_utterance: parsed 'yes' to yes
parse_and_ground_utterance: resulting groundings with normalized confidences: 
	yes 1.0
say_to_user_with_referents: Where should I go? {'action': 'walk'}
get_from_user requested feedback
get_from_user (raw): 'The office of Robert Brown.'
get_from_user (processed): 'the office of robert brown'
say_to_user: I haven't heard the word 'brown' before. Does it refer to properties of things, like a color, shape, or weight?
get_from_user requested feedback
get_from_user (raw): 'no'
get_from_user (processed): 'no'
parse_and_ground_utterance: parsed 'no' to no
parse_and_ground_utterance: resulting groundings with normalized confidences: 
	no 1.0
Traceback (most recent call last):
  File "main.py", line 175, in <module>
    main()
  File "main.py", line 109, in main
    perception_labels_requested)
  File "/var/www/gandalf.nmtechgroup.com/phm/grounded_dialog_agent/Agent.py", line 163, in start_action_dialog
    gprs, pr = self.parse_and_ground_utterance(ur)
  File "/var/www/gandalf.nmtechgroup.com/phm/grounded_dialog_agent/Agent.py", line 861, in parse_and_ground_utterance
    cgtr = self.call_generator_with_timeout(parse_generator, self.budget_for_parsing)
  File "/var/www/gandalf.nmtechgroup.com/phm/grounded_dialog_agent/Agent.py", line 1556, in call_generator_with_timeout
    r = next(g)
  File "../tsp/CKYParser.py", line 834, in most_likely_cky_parse
    parse_tree, parse_score, new_lexicon_entries = next(parse_tree_generator)
  File "../tsp/CKYParser.py", line 953, in most_likely_reranked_cky_parse
    for curr_tree, curr_new_lex in curr_generator:
  File "../tsp/CKYParser.py", line 988, in most_likely_tree_generator
    parse_roots, parse_leaves_keys = self.form_root_from_leaves(parse_leaves, ccg_tree)
  File "../tsp/CKYParser.py", line 1154, in form_root_from_leaves
    root = self.perform_fa(l, r, top_level_call=True)
  File "../tsp/CKYParser.py", line 1994, in perform_fa
    ab.renumerate_lambdas([])
  File "../tsp/SemanticNode.py", line 101, in renumerate_lambdas
    c.renumerate_lambdas(lambdas[:])
  File "../tsp/SemanticNode.py", line 101, in renumerate_lambdas
    c.renumerate_lambdas(lambdas[:])
  File "../tsp/SemanticNode.py", line 101, in renumerate_lambdas
    c.renumerate_lambdas(lambdas[:])
  File "../tsp/SemanticNode.py", line 101, in renumerate_lambdas
    c.renumerate_lambdas(lambdas[:])
  File "../tsp/SemanticNode.py", line 101, in renumerate_lambdas
    c.renumerate_lambdas(lambdas[:])
  File "../tsp/SemanticNode.py", line 98, in renumerate_lambdas
    self.lambda_name = lambdas.index(self.lambda_name) + 1
ValueError: 5 is not in list
